.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+updates:
+  - package-ecosystem: "yarn"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
what do you think about running daily the dependabot?

Reply:
Sounds like a good idea. I'll make an update within the next hour.